tests/settings.py

```diff
@@ -68,7 +68,9 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [
+            os.path.join(os.path.dirname(BASE_DIR), 'openwisp_radius', 'templates')
```

Comment:
this should not be necessary, if you have to add this here, something is wrong.
This configuration here is used to allow to customize templates at project level, while we are working at reusable-app level.

Reply:
I tried doing it without writing this here but the default template is not overwritten by any of the directory structures I tried at app level. What I think the bug is the template loaders search for the app level directories in the order INSTALLED APP. And 'django.contrib.admin' precedes openwisp_radius so it's using that template. Though I'm not so sure of this.
What should I do?